backend/src/main/java/com/khnure/timetable/synchronizer/controller/CalendarController.java

```diff
@@ -0,0 +1,41 @@
+package com.khnure.timetable.synchronizer.controller;
+
+import com.khnure.timetable.synchronizer.dto.ScheduleDto;
+import com.khnure.timetable.synchronizer.service.NureScheduleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+@RequiredArgsConstructor
+public class CalendarController {
```

Comment:
Why we have a name "Calendar", if the Controller returns data from NURE database (Mindenit API in our case)? Please, rename it like to "NureScheduleController".